src/scraper.py

```diff
@@ -1,20 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 16.01.2019
```

Comment:
I think its enough to have this versioning information, like date and author, contained in git.

Reply:
I personally love the information in the header so that I do not need to use any other tool than a text editor. I am going to leave it in the file.